.changeset/purple-books-guess.md

```diff
@@ -0,0 +1,5 @@
+---
+'@sveltejs/kit': patch
+---
+
+fix: correct ResolveOptions['preload'] inline documentation
```

Comment:
I usually go with `docs:` for doc fixes so that people can more easily differentiate from bug fixes
```suggestion
docs: correct `ResolveOptions['preload']` inline documentation
```